app/src/main/java/org/listenbrainz/android/ui/components/dialogs/BaseDialog.kt

```diff
@@ -8,6 +8,8 @@ import androidx.compose.foundation.layout.ColumnScope
 import androidx.compose.foundation.layout.fillMaxWidth
 import androidx.compose.foundation.layout.padding
 import androidx.compose.foundation.layout.sizeIn
+import androidx.compose.foundation.rememberScrollState
```

Comment:
@07jasjeet, please remember that all these components should be moved to the design system soon. So keep them as generic as possible.

Reply:
Understood.